Add files via upload

MatsAndrew opened the pull request:
Не успеваю доделать последнее задание со звёздочкой. С остальным более или менее разобрался

MatsAndrew committed:
Add files via upload

Не успеваю доделать последнее задание со звёздочкой

legacy72 commented on task_1.py:
```diff
@@ -0,0 +1,10 @@
+list_1 = [1, False, 3.5, 'name', None]
+
+
+def el_type(el):
```
Уже функции решил использовать)

legacy72 commented on task_2.py:
```diff
@@ -0,0 +1,9 @@
+a = 0
+for i in range(int(len(lst)/2)):
+    lst[a], lst[a+1] = lst[a+1], lst[a]
+    a += 2
```
Можно было просто добавить шаг в цикле `for i in range(0, len(my_list) - 1, 2)`

legacy72 commented on task_3.py:
```diff
@@ -0,0 +1,23 @@
+# task_3
+
+seasons_lst = ['Зима', 'Весна', 'Лето', 'Осень']
+seasons_dict = {1 : 'Зима', 2 : 'Весна', 3 : 'Лето', 4 : 'Осень'}
```
Чтобы не писать условия, можно было сразу подготовить словарь, в котором ключами будут номера месяца, а значениями времена года. Тогда можно было бы сразу обращаться по ключу к нужному значению

legacy72 commented on task_3.py:
```diff
@@ -0,0 +1,23 @@
+# task_3
+
+seasons_lst = ['Зима', 'Весна', 'Лето', 'Осень']
```
Тут можно было просто создать список и обращаться по индексу к нужному времени года `['зима', 'зима', 'весна', 'весна', 'весна', 'лето', 'лето', 'лето', 'осень', 'осень', 'осень', 'зима']`

legacy72 commented on task_4.py:
```diff
@@ -0,0 +1,8 @@
+# task_4
+
+my_str = input("Введите строку: ")
+a = my_str.split(' ')
```
Можно писать просто `.split()`, так как сплит по пробелам уже установлен по умолчанию в этом методе

legacy72 commented on task_4.py:
```diff
@@ -0,0 +1,8 @@
+my_str = input("Введите строку: ")
+a = my_str.split(' ')
+for i, el in enumerate(a, 1):
+    if len(el) >= 10:
```
Условия тут излишни, можно всегда обрезать `[:10]` и если будет меньше, то питон это распознает и не будет ошибки

legacy72 commented on task_4.py:
```diff
@@ -0,0 +1,8 @@
+a = my_str.split(' ')
+for i, el in enumerate(a, 1):
+    if len(el) >= 10:
+        el = el[0:10]
```
Можно не указывать нолик, он и так ставится по умолчанию. `[:10]`

legacy72 commented on task_5.py:
```diff
@@ -0,0 +1,18 @@
+#task_5
+
+number = int(input("Введите число: "))
+my_list = [5, 4, 3, 5, 2]
```
Почему список неотсортированный?

MatsAndrew replied:
Немного не понял как его надо было отсортировать

legacy72 replied:
В неубывающей последовательности, т.е. `[5, 5, 4, 3, 2]`

MatsAndrew committed:
Lesson_2_task_6

Доделал 6. За основу взял ваш пример) Конечно, начинаешь исправлять с вой код, когда видишь идеальный. Буду сдавать задания раньше, чтообы увидеть свой уровень. Спасибо, за разъяснения.

MatsAndrew committed:
Add files via upload
